aries-site/src/components/content/ComingSoon.js

```diff
@@ -18,32 +18,22 @@ export const ComingSoon = () => {
           This page appears to be empty, but content will be coming soon!
         </Text>
       </ContentSection>
-      <ContentSection lastSection>
+      <ContentSection>
         <Box direction="row" gap="small">
           <Text size={textSize} weight="bold">
             HPE
           </Text>
           <Text size={textSize}>Design System</Text>
```

Comment:
Do you think we should be referring to "Carte" now? I don't know how all in we are ready to be. (pun unintentional). If so,
```suggestion
          <Text size={textSize}>Carte Design System</Text>
```

Reply:
Let's do it to be consistent. Easy to remove if we change it later on.